slides/_licenses.qmd

```diff
@@ -0,0 +1,34 @@
+ 
+## How to choose a license
+ 
+ 
```

Comment:
Do we want to briefly mention the copyleft - permissive spectrum, and the relative attributes of each?

Reply:
I added a graphic -- could you check whether you think that's enough?

Reply:
I think that's good, choosealicense also shows this which we link to.
We can show the graphic and describe where key licenses (MIT, GPL) are.

This is much easier than my previous presentations where I tried to describe it. Why didn't I think of using a picture 😅 🤦